spec/models/candidate_spec.rb

```diff
@@ -462,7 +462,7 @@
     end
 
     context 'with no published_preferences' do
-      it 'returs true' do
+      it 'returns true' do
```

Comment:
In order to make this test pass, the method has to be: 
`published_preferences.empty? || published_preferences.last.training_locations_anywhere?`

rather than 
`published_preferences.last&.training_locations_anywhere?`

Do we really want to return true if there are no published preferences? I'm not sure it makes a lot of sense. If they haven't published their preferences, surely we don't want to say 'yes, they would be happy to go anywhere?'

Reply:
@CatalinVoineag and I decided this method wasn't actually necessary at all and deleted it.